docs/docs/quickstart-inference.md

```diff
@@ -0,0 +1,283 @@
+        
+        # Score the output
+        score = accuracy_scorer(
+            expected=example.get("expected", ""),
+            output=output,
+            expected_one_of=example.get("expected_one_of")
+        )
+        scores.append(score["score"])
+    
+    return {
+        "model": model.model_name,
+        "accuracy": sum(scores) / len(scores),
+        "total_correct": sum(scores),
+        "total_examples": len(dataset)
+    }
+
+# Run evaluation
+results = evaluate_model(llama_model, dataset)
+print(f"Model: {results['model']}")
+print(f"Accuracy: {results['accuracy']:.2%}")
+print(f"Correct: {results['total_correct']}/{results['total_examples']}")
+```
+
+## Step 6: View your results in Weave
+
+After running these examples, you'll see links in your terminal. Click any link to view traces in the Weave UI.
+
+In the Weave UI, you can:
+- See a timeline of all your LLM calls
+- Inspect inputs and outputs for each operation
+- View token usage and estimated costs
+- Analyze latency and performance metrics
+- Compare different runs
+
+![Weave Traces View](guides/tools/imgs/image.png)
+
+## Available models
+
+W&B Inference provides access to many models including Llama 3.1 8B for general-purpose tasks, Llama 3.3 70B for complex reasoning, DeepSeek V3 for language processing, and Qwen3 235B for mathematical and structured reasoning. 
+
+For a complete list of available models with specifications, context windows, pricing, and usage limits, see the [Available Models section](guides/tools/inference.md#available-models) in the W&B Inference reference documentation.
+
+## Next steps
+
+After completing this guide:
+
+- **Use the Playground**: [Try models interactively](guides/tools/playground.md#access-the-playground) in the Weave Playground
+- **Build evaluations**: Learn about [systematic evaluation](guides/core-types/evaluations.md) of your LLM applications
+- **Try other integrations**: Weave works with [OpenAI, Anthropic, and many more](guides/integrations/index.md)
+- **Deploy your app**: Use [Weave Serve](guides/tools/serve.md) to deploy your applications
+
+## Troubleshooting
+
+<details>
+<summary>Authentication errors</summary>
+
+If you get authentication errors:
+1. Verify you have a valid W&B account
+2. Check that you're using the correct API key from [wandb.ai/authorize](https://wandb.ai/authorize)
+3. Ensure your project name follows the format `team-name/project-name`
+
+</details>
+
+<details>
+<summary>Rate limit errors</summary>
+
+W&B Inference has concurrency limits per project. If you hit rate limits:
+- Reduce the number of concurrent requests
+- Add delays between calls
+- Consider upgrading your plan for higher limits
+
+</details>
+
+<details>
+<summary>Running out of credits</summary>
+
+The free tier includes limited credits. To continue using W&B Inference:
+- Free users: [Upgrade to Pro](https://wandb.ai/subscriptions)
+- Pro users: Usage is billed monthly after free credits (default cap: $6,000/month)
```

Comment:
```suggestion
- Free users: [Upgrade to Pro](https://wandb.ai/subscriptions).
- Pro users: Usage is billed monthly after free credits.
```
I think we try hard to avoid having specific pricing details in docs, in favor of the main pricing page.

Reply:
Yeah, removed the details here.